readme.md

```diff
@@ -24,42 +24,61 @@ or in the preview below.
 
 The simplest use of the library in both platforms could be done as follows:
 
-If on the server, install it through npm:
+### Node.js
+
+If on the server, install it through [npm](https://www.npmjs.com/):
 
 ```bash
 npm install --save logdown
 ```
 
-In the browser you can install it through [bower](http://bower.io).
+```js
+var Logdown = require('logdown')
+var logger = new Logdown({prefix: 'foo'})
+```
+
+### Browser
+
+In the browser you can install it through [Bower](http://bower.io).
 
 ```bash
 bower install logdown
 ```
 
-You can also use the lib in the browser in the same way as in the server if you
-use [Browserify](http://browserify.org/). Or you can just download it
-[here](https://github.com/caiogondim/logdown.js/archive/master.zip) and put the
-`dist/index.js` file in your public folder.
-
 ```js
-// In the browser
 var logger = new Logdown({prefix: 'foo'})
 ```
 
-And then use it.
+### SystemJS
+
+Using the dynamic module loader [SystemJS](https://github.com/systemjs/systemjs), Logdown can be loaded as a CommonJS module.
 
 ```js
-// In the server-side or client-side with Browserify
-var Logdown = require('logdown')
-var logger = new Logdown({prefix: 'foo'})
+SystemJS.config({
+  map: {
+    'logdown': 'bower_components/logdown/dist/index.js'
+  },
+  packages: {
+    'logdown': {format: 'cjs'}
+  }
+});
 ```
 
-It is highly recommended to use a prefix for your instance, this way you get
-a nice prefixed message on console and it is possible to silence instances
-based on the prefix name, as we will see after.
+```js
+System.import('logdown').then(function(Logdown) {
+  var logger = new Logdown({prefix: 'foo'}
+});
+```
+
+### Other
+
+You can also use the lib in the browser in the same way as in the server if you use [Browserify](http://browserify.org/). Or you can just download it [here](https://github.com/caiogondim/logdown.js/archive/master.zip) and put the `dist/index.js` file in your public folder.
```

Comment:
I assume this is not related to the commit...?

The reason I put phrases in more than one line is that I don't like code and text to be bigger than 80 columns
Improves readability and avoids nesting